cmd/examples/nse/nse.go

```diff
@@ -185,10 +185,10 @@ func main() {
 		Metadata: &common.Metadata{
 			Name: "gold-net-channel-1",
 		},
-		NseProviderName:    podName,
-		NetworkServiceName: networkServiceName,
+		Nseprovidername:    podName,
```

Comment:
Why changing the Camel style?

Reply:
Weird, that sounds more like a bug in openapi-gen, since Camel Style is pretty much a standard for go programs.

Reply:
See my reply to your other comment. I'm open to ideas there, so let me know if you have any ideas to work around this. Otherwise, to me it's not a huge deal.